charts/drax/values.yaml

```diff
@@ -544,6 +543,18 @@ grafana:
             organization: accelleran
             defaultBucket: default
             tlsSkipVerify: true
+        - name: InfluxDB2
```

Comment:
```suggestion
       - name: InfluxDB2
          type: influxdb
          access: proxy
          url: "http://{{ $.Release.Name }}-influxdb2.{{ $.Release.Namespace }}:80"
          uid: JOSE3g9KVb
          secureJsonData:
            token: ${INFLUXDB2_ADMIN_TOKEN}
          jsonData:
            version: Flux
            organization: accelleran
            defaultBucket: default
            tlsSkipVerify: true
```

Reply:
Then we also need to add the INFLUXDB2_ADMIN_TOKEN under envValueFrom

```
INFLUXDB2_ADMIN_TOKEN:
      secretKeyRef:
        name: "{{ $.Release.Name }}-influxdb2-auth"
        key: admin-token
```

Reply:
These were done in our interactive session